tools/install.sh

```diff
@@ -263,10 +269,29 @@ elif [ "$LINUX_VERSION" == "BackBox" ] || \
 	 [ "$LINUX_VERSION" == "Ubuntu"  ]; then
 	apt-get install -y libfreetype6-dev python-dev python-pip
 	if [ -z "$KING_PHISHER_SKIP_CLIENT" ]; then
-		apt-get install -y gir1.2-gtk-3.0 gir1.2-gtksource-3.0 gir1.2-vte-2.90 \
+		apt-get install -y gir1.2-gtk-3.0 gir1.2-gtksource-3.0 \
+			gir1.2-webkit-3.0 python-cairo libgeos++-dev \
+			libgtk-3-dev libpq-dev python-gi python-gi-cairo libpq-dev \
+			python-gobject python-gobject-dev python-paramiko pkg-config
+		if [ $? -eq 0 ]; then
```

Comment:
This logic should be inverted to check for the failure condition, print a simple error message and bail. The success logic should be the same level of indentation as this check. I also would like to avoid having all of the packages listed a second time in the error message. Telling the user an error occurred in this step would already be more than currently exists.